VirtoCommerce.PricingModule.Web/Scripts/blades/item/item-pricelists-list.js

```diff
@@ -1,49 +1,49 @@
-﻿angular.module('virtoCommerce.pricingModule')
+angular.module('virtoCommerce.pricingModule')
```

Comment:
Please revert this file, if there are no meaningful changes (only formatting).

Reply:
it`s done.